Pollo/Supporting/Constants.swift

```diff
@@ -117,8 +117,8 @@ struct StringConstants {
 }
 
 struct IntegerConstants {
-    static let maxOptionsForAdminMC = 4
-    static let maxOptionsForMemberMC = 4
+    static let maxOptionsForAdminMC = 5
```

Comment:
Why do we constrain the number of options to a hard variable? Shouldn't the card just grow until its the max height based on the boundary constraints of the screen?